src/builder.rs

```diff
@@ -564,6 +564,7 @@ impl ContainerOptionsBuilder {
         self
     }
 
+    /// Specify any bind mounts, taking the form of `/some/host/path:/some/container/path`
```

Comment:
I think `volumes` is not restricted to bind mounts. You can use also use named volumes. For example, you could use `volumes(vec!["myvol2:/app"])` (see [here](https://docs.docker.com/storage/volumes/))